### DISCUSSION TITLE
navigation voice instructions error TTS

### DISCUSSION THREAD

**Discussion by letkan** [osmandapp/OsmAnd#11858] [Maps Related]
In roundabout with 3 exits where I need to take the first one (and map shows it correctly!) the voice says "take the 3rd exit". On another with 4 exits where I need to exit  the 2nd one, I'm being told to take the 3rd one . How can I fix this?

_Mentiond here also: https://github.com/osmandapp/OsmAnd/issues/9641#issuecomment-678118922_

**Comment by sonora** [comment]
Can you please state examples, I believe this needs to be viewed in connection with how the roundabouts are actually mapped. Thx!

**Comment by letkan** [comment]
> Can you please state examples, I believe this needs to be viewed in connection with how the roundabouts are actually mapped. Thx!

Ok, I remember they were 2 roundabouts on a route I do frequently but next time I'll make a note of the exact location and pass it along.

**Comment by mikehgentry** [comment]
Here's one that seems to consistently cause trouble, at 51.09449N, 0.20077W

E.g. navigate from Coachman's Drive, Broadfield, to Tollgate Hill, Crawley, and it should tell you verbally to take the 3rd exit and show a 3 icon, but clearly want you to take the 2nd.

Edit: sorry, this was doing it consistently last night, and now isn't. No idea.
